src/main/java/uk/gov/hmcts/divorce/common/service/task/GenerateAosResponseLetterDocument.java

```diff
@@ -89,6 +98,31 @@ public CaseDetails<CaseData, State> apply(CaseDetails<CaseData, State> caseDetai
                             AOS_RESPONSE_LETTER_DOCUMENT_NAME
                         );
                     }
+                } else {
+                    if (caseData.getApplicant1().isRepresented()) {
+                        log.info("Generating aos response (undefended) JS letter pdf, D10 and D84 forms for case id: {}", caseId);
+
+                        generateD10Form.apply(caseDetails);
+                        generateD84Form.generateD84Document(caseData, caseId);
+
+                        generateCoversheet.generateCoversheet(
+                                caseData,
+                                caseId,
+                                COVERSHEET_APPLICANT2_SOLICITOR,
```

Comment:
(We're checking if app1 is represented, then using the app2 template - thought it was worth double checking)

Reply:
The solicitor coversheet is generic so it is technically correct to use it. However we have updated the code to make solicitor coversheet name generic too in one of the branches and this will eventually be overridden when that one is merged